common/config/rush/pnpm-lock.yaml

```diff
@@ -509,7 +509,9 @@ importers:
       '@itwin/ecschema-metadata': ^4.1.0
       '@itwin/eslint-plugin': ^3.7.7
       '@itwin/imodel-components-react': ^4.1.0
-      '@itwin/imodels-client-management': ^3.1.0
+      '@itwin/imodels-access-common': ^4.1.2
```

Comment:
Any reason why this specific version is picked? Does 4.1.0 have what you need? Maybe even the 4.0.0 has what you need?

Reply:
I don't think it should be an issue either way? This is just a local dependency. If this would be a peer dependency yes. Package managers can handle multiple versions.

Reply:
Yes, it's not an issue. I added this comment and thought that if users have 4.x.x in their project this will not cause any breaking changes.